jekyll/_cci2/release/manage-releases.adoc

```diff
@@ -0,0 +1,189 @@
+* For a Kubernetes Deployment:
++
+[,shell]
+----
+kubectl rollout undo deployment/<deployment-name> -n <namespace> --to-revision=<revision-number>
+----
+
+
+[#scale-version]
+== Scale version
+
+The _scale version_ command allows you to scale up or down a currently live version of a component.
+
+image::../../img/docs/releases/scale-version.png[Screenshot showing the scale version button]
+
+[#scale-steps]
+=== Steps
+
+. Select btn:[Releases] in the CircleCI web app sidebar
+. Select the **Components** tab
+. Scroll to find your component a click to enter the component view
+. Find the active version that you would like to scale, and click the scale icon
+. Read through the notes associated with this command in the popup and to proceed type `SCALE` and click btn:[Scale]
+
+CAUTION: Any Horizontal Pod Autoscalers (HPAs) defined for this component will be deleted as a result of this action
+
+[#scale-notes]
+=== Notes
+
+Executing the _scale version_ command is the equivalent of the following:
+
+* If you are using Argo Rollouts:
++
+[,shell]
+----
+kubectl patch rollout <rollout_name> -n <namespace> --type='json' -p='[{"op": "replace", "path": "/spec/replicas", "value": <number_of_replicas>}]'
+----
+
+* For a Kubernetes Deployment:
++
+[,shell]
+----
+kubectl patch deployment <deployment_name> -n <namespace> --type='json' -p='[{"op": "replace", "path": "/spec/replicas", "value": <number_of_replicas>}]'
+----
+
+[#restart-version]
+== Restart version
+
+The _restart version_ command allows you to restart a currently live version of a component.
+
+image::../../img/docs/releases/restart-version.png[Screenshot showing the restart version button]
+
+[#start-steps]
+=== Steps
+
+. Select btn:[Releases] in the CircleCI web app sidebar
+. Select the **Components** tab
+. Scroll to find your component a click to enter the component view
+. Find the active version that you would like to restart, and click the restart icon
+. Read the note associated with this command in the popup and to proceed type `RESTART` and click btn:[RESTART]
+
+[#restart-notes]
+=== Notes
+
+Executing the _restart version_ command is the equivalent of the following:
+
+* If you are using Argo Rollouts:
++
+[,shell]
+----
+kubectl patch rollout my-rollout -n my-namespace --type='json' -p='[{"op": "replace", "path": "/spec/restartAt", "value": <timestamp>}]'
+----
+
+* For a Kubernetes Deployment:
++
+[,shell]
+----
+kubectl rollout restart deployment/<deployment-name> -n <namespace>
+----
+
+
+[#retry-release]
+== Retry release
+
+NOTE: You can use this command if you are using link:https://argoproj.github.io/argo-rollouts/[Argo Rollouts]
+
+The _retry release_ command is available to run an link:https://argo-rollouts.readthedocs.io/en/latest/generated/kubectl-argo-rollouts/kubectl-argo-rollouts_retry/[Argo Rollout retry]. You may want to use this command if your release has entered a xref:releases-overview#release-status[failed] state.
+
+[#promote]
+== Promote
```

Comment:
@avalcepina this section needs checking especially as I am not sure I have the descriptions right for skipping "steps"

Reply:
@ryanbarkley I think I added this after your initial review too so would be interested to hear your comments if any

Reply:
It looks correct to me